apps/web/components/booking/BookingListItem.tsx

```diff
@@ -508,72 +511,94 @@ function BookingListItem(booking: BookingItemProps) {
-                  <MeetingTimeInTimezones
-                    timeFormat={userTimeFormat}
-                    userTimezone={userTimeZone}
-                    startTime={booking.startTime}
-                    endTime={booking.endTime}
-                    attendees={booking.attendees}
-                  />
+      <tr data-testid="booking-item" className="hover:bg-muted group transition ">
+        <div className="flex flex-col sm:flex-row">
+          <td className="hidden align-top ltr:pl-3 rtl:pr-6 sm:table-cell sm:min-w-[12rem]">
+            <div className="flex h-full items-center">
+              {eventTypeColor && (
+                <div className="h-[70%] w-0.5" style={{ backgroundColor: eventTypeColor }} />
+              )}
+              <Link href={bookingLink} className="ml-3">
+                <div className="cursor-pointer py-4">
+                  <div className="text-emphasis text-sm leading-6">{startTime}</div>
+                  <div className="text-subtle text-sm">
+                    {formatTime(booking.startTime, userTimeFormat, userTimeZone)} -{" "}
+                    {formatTime(booking.endTime, userTimeFormat, userTimeZone)}
+                    <MeetingTimeInTimezones
+                      timeFormat={userTimeFormat}
+                      userTimezone={userTimeZone}
+                      startTime={booking.startTime}
+                      endTime={booking.endTime}
+                      attendees={booking.attendees}
+                    />
+                  </div>
+                  {!isPending && (
+                    <div>
+                      {(provider?.label || locationToDisplay?.startsWith("https://")) &&
+                        locationToDisplay.startsWith("http") && (
+                          <a
+                            href={locationToDisplay}
+                            onClick={(e) => e.stopPropagation()}
+                            target="_blank"
+                            title={locationToDisplay}
+                            rel="noreferrer"
+                            className="text-sm leading-6 text-blue-600 hover:underline dark:text-blue-400">
+                            <div className="flex items-center gap-2">
+                              {provider?.iconUrl && (
+                                <img
+                                  src={provider.iconUrl}
+                                  className="h-4 w-4 rounded-sm"
+                                  alt={`${provider?.label} logo`}
+                                />
+                              )}
+                              {provider?.label
+                                ? t("join_event_location", { eventLocationType: provider?.label })
+                                : t("join_meeting")}
+                            </div>
+                          </a>
+                        )}
+                    </div>
+                  )}
                 </div>
-                {!isPending && (
```

Comment:
Had to move the badges to a separate component when the screen size isn't mobile.